src/GraphView.ts

```diff
@@ -276,20 +267,22 @@ export default class GraphView {
         this.p.strokeWeight(0.5);
         this.p.fill(0);
 
-        this.p.text("O", this.canvasWidth/2 - 15, this.canvasHeight/2 + 15);
-        this.p.text("x", this.canvasWidth - this.PADDING, this.canvasHeight/2 + 15);
-        this.p.text("y", this.canvasWidth/2 + 5, this.PADDING);
+        this.p.text("O", this.canvasProperties.centerPx[0] - 15, this.canvasProperties.centerPx[1] + 15);
+        this.p.text("x", this.canvasProperties.centerPx[0] + this.canvasProperties.axisLengthPx/2 - 2 * this.PADDING, this.canvasProperties.centerPx[1] + 15);
```

Comment:
This line seems to double the previous PADDING value for x? Was that intentional?